src/chrome/content/rules/OpenITP.org.xml

```diff
@@ -1,4 +1,11 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://openitp.org/ => https://openitp.org/: None
```

Comment:
'None' is not very descriptive. Perhaps it's the mixed stylesheet from fonts.googleapis.com?